### PR TITLE
Metadata update

### PR DESCRIPTION
**Pull request by anonymoussprocket**

### CODE REVIEW

File: src/main/resources/metadata/tezos.carthagenet.conf [hunk 1]
```diff
@@ -303,8 +303,8 @@
       }
     }
     big_maps {
-      display-name-plural: "Big Maps"
-      display-name: "Big Map"
+      display-name-plural: "Map Definitions"
```
Comment (vishakh) [inline src/main/resources/metadata/tezos.carthagenet.conf]: I feel this name is too abstract. There can be different kinds of maps.

Comment (anonymoussprocket) [reply src/main/resources/metadata/tezos.carthagenet.conf]: In the current context, I think this is a better way to describe them. We can also add a "description" field if you'd like.

Comment (vishakh) [reply src/main/resources/metadata/tezos.carthagenet.conf]: A description would be good. Also, this branch now has a merge conflict.

Comment (anonymoussprocket) [reply src/main/resources/metadata/tezos.carthagenet.conf]: the conflict is due to a file that should die

Comment (vishakh) [reply src/main/resources/metadata/tezos.carthagenet.conf]: Backwards compatibility should not be so trivially broken. A simple pull from `master` should sort it out so we can merge this PR.

**Comment by vishakh** [comment]
Assuming this is abandoned.

**Pull request closed without merging**